pkg/machine/fedora.go

```diff
@@ -0,0 +1,129 @@
+// +build amd64 arm64
+
+package machine
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	githubURL      = "http://github.com/fedora-cloud/docker-brew-fedora/"
+	defaultRelease = "35"
```

Comment:
The two should probably be configurable (e.g., via `man containers.conf` c/common/pkg/config). @rhatdan WDYT?

Reply:
Agreed